provision/vvv-init.sh

```
@@ -15,7 +15,7 @@ try_npm_install() {
     echo " * Removing the node modules folder"
     rm -rf node_modules
     echo " * Clearing npm cache"
-    noroot npm cache clean --force
+    noroot npm cache clean --force &> /dev/null
```

This would silence any errors too though?

---

No will silence this:
```
default: npm
    default:  
    default: WARN
    default:  using --force I sure hope you know what you are doing.
```